p4-16/spec/P4-16-spec.mdk

```diff
@@ -4641,8 +4639,8 @@ parser Tcp_option_parser(packet_in b, out Tcp_option_stack vec) {
 }
 ~End P4Example
 
-Similar to headers, the size of a header union can be determined at
-compile-time (Section [#sec-minsizeinbits]).
+Similar to headers, the size of a header union is a compile-time known
```

Comment:
Why is it not local compile-time known? That would allow putting it e.g. in `bit<...>` (I don't know why one would want that).

Reply:
Fixed. Good catch!